README.md

```diff
@@ -41,6 +41,9 @@ python3 -m driver.main --aws-region <region> --db-identifier <db-identifier-with
 
 If you are using postgres you will need to set the `POSTGRES_OTTERTUNE_DB_NAME` env variable if you are using a postgres database.
 
+## Permissions
+To collect metrics from the database, you need to [configure database settings](https://docs.ottertune.com/info/connect-your-database-to-ottertune/add-database#configure-database-settings), and [grant permissions](https://docs.ottertune.com/info/connect-your-database-to-ottertune/add-database/guarded-public-access#create-the-database-user-and-grant-permissions-1) to the database user.
+
```

Comment:
should we add a `running from source` section instead of putting this in the `Development` section?

Reply:
I think we also need such permissions during the development. Do we want to merge this first and add a new `Running from source` section later ?

Reply:
sounds good to me